python/xi_plugin/host.py

```diff
@@ -99,6 +98,17 @@ def shutdown(self, peer, **params):
         peer.done = True
         self.plugin.shutdown()
 
+    def other_cmd(self, peer, method, params):
+        '''Unknown RPC, presumably a custom command provided by the plugin.'''
```

Comment:
when we receive the rpc (https://github.com/google/xi-editor/pull/366/files#diff-761f7b0ecb719bd1378e85f32c16fabbR77) we check to see if the method is one the host already implements; only if we don't know the command we call this. It is an expectation of the current architecture that any method that is explicitly part of the plugin protocol should be handled by the host.

It is currently possible to declare a custom command that shares a method name with a plugin protocol method; in the future we can check for this case when we parse plugin manifests.